bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
+
 # Get cloud provider
 _setArgs(){
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--ci")
+        ci="yes"
+        ;;
+
       "--azure")
-       cloudProvider="azure"
-       emulator="azurite"
-       devServer="http://localhost:10000"
-       export STORAGE_SERVICE_NAME="azure-blob"
-       ;;
+        readonly CLOUD_PROVIDER="azure"
```

Comment:
optional: Would be nice to check if azure and aws are both set and issue an error.